Connectors/CSi/Speckle.Connectors.CSiShared/cPlugin.cs

```diff
@@ -0,0 +1,63 @@
+// NOTE: Plugin entry point must match the assembly name, otherwise hits you with a "Not found" error when loading plugin
+// TODO: Move ETABS implementation to csproj as part of CNX-835 and/or CNX-828
+namespace Speckle.Connectors.ETABS22;
```

Comment:
wrong namespace?

Reply:
Temp solution, that's why I marked as TODO. ETABS expects cPlugin and Form1 namespace match assembly name. When it was under CSiShared, it gave the "Not Found" error. Will make it more elegant in next series of Ticket(s)

Reply:
very annoying